pack/return/rttcu.json

```diff
@@ -1,52 +1,254 @@
+        "pack_code": "rttcu",
+        "position": 10,
+        "quantity": 2,
+        "skill_wild": 2,
+        "text": "Fast. Play only during your turn.\nUntil the end of your turn, set the base value of each of your skills to 5, or the base value of one of your skills to 7.",
+        "traits": "Spirit.",
+        "type_code": "event",
+        "xp": 3
+    },
+    {
+        "code": "54011",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "neutral",
+        "illustrator": "Robert Laskey",
+        "name": "The Fool • 0",
+        "pack_code": "rttcu",
+        "position": 11,
+        "quantity": 2,
+        "slot": "Tarot",
+        "subname": "Unlimited Potential",
+        "text": "[reaction] When you play a card, exhaust The Fool • 0: Reduce that card's cost by 1.\n[reaction] When the game begins, if The Fool • 0 is in your opening hand: Put it into play.",
+        "traits": "Tarot.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "54012",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "neutral",
+        "illustrator": "Drazenka Kimpel",
+        "name": "Moon Pendant",
+        "pack_code": "rttcu",
+        "position": 12,
+        "quantity": 2,
+        "slot": "Accessory",
+        "text": "You have 1 additional tarot slot.\nEach non-weakness [[Tarot]] asset in your hand or committed to a skill test gains [wild] [wild].\n[reaction] When the game begins, if Moon Pendant is in your opening hand: Put it into play.",
+        "traits": "Item. Charm.",
+        "type_code": "asset",
+        "xp": 2
+    },
+    {
+        "code": "54013",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Allan Bednar",
+        "name": "Moon Pendant",
+        "pack_code": "rttcu",
+        "permanent": true,
+        "position": 13,
+        "quantity": 1,
+        "text": "Permanent. Limit 1 per investigator.\nBefore setting up each game, draw 3 random cards from the tarot deck and choose 1. Place the chosen card in front of you, in the upright position. Each other investigator ignores its effects. Shuffle the rest back into the tarot deck.",
+        "traits": "Blessed.",
+        "type_code": "asset",
+        "xp": 4
+    },
+    {
+        "code": "54014",
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Marco Caradonna",
+        "name": "Damned",
+        "pack_code": "rttcu",
+        "permanent": true,
+        "position": 14,
+        "quantity": 1,
+        "subtype_code": "basicweakness",
+        "text": "Permanent.\nBefore setting up each game, draw a random card from the tarot deck and place the chosen card in front of you, in the reverse position. Each other investigator ignores its effects.",
+        "traits": "Curse. Omen.",
+        "type_code": "treachery"
+    },
+    {
+        "code": "54015",
+        "cost": 3,
+        "deck_limit": 1,
```

Comment:
Limit is 2 I think, because quantity is 2.

Reply:
Good to know, that's unfortunate cause this card looks mean.